ddtrace/settings/__init__.py

```diff
@@ -1,12 +1,17 @@
-from .._hooks import Hooks
-from .exceptions import ConfigException
-from .http import HttpConfig
-from .integration import IntegrationConfig
```

Comment:
You can't get rid of these imports. User's should still be able to import Hooks/ConfigException/HttpConfig from `ddtrace.settings`. All we are doing here is logging is warning when this happens. 

We can remove these imports in v4.0 (since this is a breaking change